mailpoet/tests/integration/Cron/Workers/SendingQueue/SendingQueueTest.php

```diff
@@ -1600,7 +1632,7 @@ private function getSendingQueueWorker($mailerMock = null, $authorizedEmailContr
       $this->sendingQueuesRepository,
       $this->entityManager,
       $this->statisticsNewslettersRepository,
-      $authorizedEmailControllerMock ?? $this->authorizedEmailsController
+      $authorizedEmailControllerMock ?? $this->authorizedEmailsController,
```

Comment:
Why did you add a comma here? I think it is not a big issue, but it can cause failing tests on lower PHP versions.

Reply:
Aha. Nice catch! It was accidental. I added one more argument, but then I decided it was not a good idea, so I deleted it and I forgot the comma. Trailing comma should be [fine for PHP7.3+](https://php.watch/versions/7.3/func-call-trailing-comma) and we use PHP7.4 as the oldest one so this should be fine even for nightly tests. So I guess it is not a big deal, right?

Reply:
Ah, right! I thought it was from 8.0, but I got it mixed up with [trailing comma in function declarations](https://php.watch/versions/8.0/trailing-comma-parameter-use-list).